src/ontology/mondo-edit.obo

```diff
@@ -560579,8 +560579,11 @@ id: MONDO:0971004
 name: amyloidosis, hereditary systemic 1
 subset: inferred_rare
 subset: rare
+xref: MEDGEN:414031 {source="MONDO:equivalentTo"}
 xref: OMIM:105210 {source="MONDO:equivalentTo"}
 is_a: MONDO:0007100 {source="OMIM:105210", source="https://orcid.org/0000-0002-4142-7154"} ! familial amyloid neuropathy
+intersection_of: MONDO:0007100 {source="https://orcid.org/0000-0002-0587-4693"} ! familial amyloid neuropathy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12405 {source="https://orcid.org/0000-0002-0587-4693"} ! TTR
```

Comment:
We do not add sources to "equivalentTo" (ie logical definition). Please remove it here.
This is one of the Mondo idiosyncrasies.

Reply:
Comment addressed